vm/actor/src/builtin/miner/policy.rs

```diff
@@ -124,10 +130,9 @@ pub fn max_prove_commit_duration(proof: RegisteredSealProof) -> Option<ChainEpoc
     use RegisteredSealProof::*;
     match proof {
         StackedDRG32GiBV1 | StackedDRG2KiBV1 | StackedDRG8MiBV1 | StackedDRG512MiBV1
-        | StackedDRG64GiBV1 | StackedDRG32GiBV1P1 | StackedDRG2KiBV1P1 | StackedDRG8MiBV1P1
-        | StackedDRG512MiBV1P1 | StackedDRG64GiBV1P1 => {
-            Some(EPOCHS_IN_DAY + PRE_COMMIT_CHALLENGE_DELAY)
-        }
+        | StackedDRG64GiBV1 => Some(EPOCHS_IN_DAY + PRE_COMMIT_CHALLENGE_DELAY),
+        StackedDRG32GiBV1P1 | StackedDRG64GiBV1P1 | StackedDRG512MiBV1P1 | StackedDRG8MiBV1P1
+        | StackedDRG2KiBV1P1 => Some(6 * EPOCHS_IN_DAY + PRE_COMMIT_CHALLENGE_DELAY),
```

Comment:
My eyes are getting a little cross-eyed comparing this code here with this:
https://github.com/filecoin-project/specs-actors/blob/97fc4480cbc9b7078c67276d8c3394e3423ef513/actors/builtin/miner/policy.go#L159-L163

I'm not sure I understand the logic and how it translates. Can you just take another look and let me know if these are 100% correct as far as you know?

Reply:
Double checked. Theyre good